src/curated-corpus/components/DuplicateProspectModal/DuplicateProspectModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { ApprovedCorpusItem } from '../../../api/generatedTypes';
+import { Modal, SharedFormButtons } from '../../../_shared/components';
+import { useStyles } from './DuplicateProspectModal.styles';
+
+interface DuplicateProspectModalProps {
+  /**
+   * Whether the modal is visible on the screen or not.
+   */
+  isOpen: boolean;
+
+  /**
+   * Toggle the DuplicateProspectModal to show/hide as necessary.
+   */
+  toggleModal: VoidFunction;
+
+  /**
+   * Item of type ApprovedCorpusItem.
+   */
+  approvedItem: ApprovedCorpusItem;
+}
+
+/**
+ * This component renders a simple modal that links to the curation history page of an already existing prospect.
+ */
+export const DuplicateProspectModal: React.FC<DuplicateProspectModalProps> = (
+  props
+): JSX.Element => {
+  const classes = useStyles();
+  const { isOpen, toggleModal, approvedItem } = props;
+
+  /**
+   *  on click function used by "View item curation history" button
+   *  opens the link in the new tab and toggles off the modal
+   */
+  const onClick = () => {
+    window.open(
+      `/curated-corpus/corpus/item/${approvedItem.externalId}`,
+      '_blank'
+    );
+
+    toggleModal();
+  };
```

Comment:
Do we lose state if the link opens in a new tab? I think we do. Is it important? Something to keep in mind but okay to go as is for now, I think.